app/assets/stylesheets/helpers/_text.scss

```diff
@@ -6,20 +6,20 @@
 .content-block,
 article {
   h1 {
-    @include bold-24;
+    @include govuk-font(24, $weight: bold);
     margin-top: 1.5em;
     margin-bottom: 0.75em;
   }
 
   header h1 {
-    @include heading-27;
+    @include govuk-font(27, $weight: bold);
```

Comment:
The [Design System How To Guide](https://design-system.service.gov.uk/get-started/updating-your-code/#mixins-and-variables) recommends that `govuk-font(27, $weight: bold)` is used to replace `heading-27`

Reply:
That's fine, I was raising this because it was handled one way here but another way here: https://github.com/alphagov/static/pull/2238/files#diff-1508b08fa30f83718afb89507ab05dfaR47

Can you explain the difference or do both these examples need to be consistent?

Reply:
It was handled differently in that other scenario because there is a specific `govuk-heading` size that can replace `heading-48`. As 48px is a heading size we still actively use. 

My understanding is that we no longer officially use 27px headings, which is why the DS guidance recommends replacing it with `govuk-font(27, $weight: bold)` when migrating to the DS.

Reply:
Great, thanks for the detail.